src/pyvesync/device_map.py

```diff
@@ -823,15 +823,26 @@ class ThermostatMap(DeviceMapTemplate):
     ),
     PurifierMap(
         class_name='VeSyncAir131',
-        dev_types=['LV-PUR131S', 'LV-RH131S', 'LV-RH131S-WM'],
+        dev_types=['LV-PUR131S'],
         modes=[PurifierModes.SLEEP, PurifierModes.MANUAL, PurifierModes.AUTO],
         features=[PurifierFeatures.AIR_QUALITY],
         fan_levels=list(range(1, 4)),
         device_alias='LV-PUR131S',
-        model_display='LV-PUR131S/RH131S Series',
+        model_display='LV-PUR131S Series',
         model_name='LV131S',
         setup_entry='LV-PUR131S',
     ),
+    PurifierMap(
+        class_name='VeSyncAirRH131',
+        dev_types=['LV-RH131S-WM'],
```

Comment:
The LV-RH131S is removed from above but not added here.  should it be?

Reply:
I don't believe it is necessary, but I added it back just in case.